static/js/info_overlay.js

```diff
@@ -22,99 +22,147 @@ export let toggler;
 
 const markdown_help_rows = [
     {
-        markdown: "*italic*",
+        markdown: "*" + $t({defaultMessage: "italic"}) + "*",
         usage_html: "(or <kbd>Ctrl</kbd>+<kbd>I</kbd>)",
     },
     {
-        markdown: "**bold**",
+        markdown: "**" + $t({defaultMessage: "Bold"}).toLowerCase() + "**",
         usage_html: "(or <kbd>Ctrl</kbd>+<kbd>B</kbd>)",
     },
     {
-        markdown: "~~strikethrough~~",
+        markdown: "~~" + $t({defaultMessage: "strikethrough"}) + "~~",
```

Comment:
What's the thinking around just putting `~~strikethrough~~` as the thing we offer to translators for these simple syntax cases? I feel like it might better indicate the context if we have the wrapping syntax on these.